tests/test_metric.py

```diff
@@ -216,16 +220,6 @@ def test_nspd(self, einstein_img, curie_img):
         assert po.metric.nspd(einstein_img, curie_img).requires_grad
         curie_img.requires_grad_(False)
 
-    def test_nspd2(self, einstein_img, curie_img):
-        curie_img.requires_grad_()
-        assert po.metric.nspd(einstein_img, curie_img, O=3, S=5, complex=True).requires_grad
-        curie_img.requires_grad_(False)
-
-    def test_nspd3(self, einstein_img, curie_img):
-        curie_img.requires_grad_()
-        assert po.metric.nspd(einstein_img, curie_img, O=1, S=5, complex=False).requires_grad
-        curie_img.requires_grad_(False)
-
```

Comment:
why remove these two and not `test_nspd`? I thought these were removed because `test_batch_handling` handles the testing for `nspd`, but then why leave `test_nspd` in?

Reply:
I assumed the purpose of these tests was to make sure the gradient works. Maybe I can change the names of these tests to things like `test_nlpd_grad`.

Reply:
Yes, I think that would greatly clarify things